File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
class Solution {
    
    private static final String[] NUMBER_WORDS = {
        "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"
    };

    public int solution(String s) {
        for (int i = 0; i < NUMBER_WORDS.length; i++) {
            s = s.replace(NUMBER_WORDS[i], String.valueOf(i));
        }
        return Integer.parseInt(s);
    }
}

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    numbers = ('zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine')
    for seq, number in enumerate(numbers):
        s = s.replace(number, str(seq))
    return int(s)
